backend/clubs/views.py

```diff
@@ -1230,6 +1230,38 @@ def post(self, request, *args, **kwargs):
         )
 
 
+class LastEmailInviteTestAPIView(APIView):
+    """
+    get: Return the club code, invite id and token of the last sent email invite
+    """
+
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        latest_email_invite = MembershipInvite.objects.filter(active=True).latest("created_at")
+        club_code = latest_email_invite.club.code
+        email_id = latest_email_invite.id
+        email_token = latest_email_invite.token
+
+        if request.user.email == latest_email_invite.email:
+            return Response(
+                {
+                    "code": club_code,
+                    "id": "{}".format(email_id),
+                    "token": "{}".format(email_token),
+                },
+            )
+        else:
+            return Response(
+                {
+                    "detail": "You can only access tokens for invitations that match your email",
```

Comment:
This is the smallest nitpick ever, but I like to have periods at the end of error messages that are sentences. Feel free not to change this.

Reply:
Not at all. It has been changed!